packages/start/src/client/Meta.tsx

```diff
@@ -173,6 +173,10 @@ export const Meta = ({ children }: { children?: React.ReactNode }) => {
   React[
     typeof document !== 'undefined' ? 'useLayoutEffect' : 'useEffect'
   ](() => {
+    if (router.isServer) {
```

Comment:
Actually, check for the document.

```suggestion
    if (typeof document !== 'undefined') {
```